keras_rcnn/heads/resnet.py

```diff
@@ -0,0 +1,96 @@
+import keras
+
+
+def ResHead(classes, mask=False):
+    """Resnet heads as in Mask R-CNN."""
+    def f(x):
+        if keras.backend.image_data_format() == "channels_last":
+            channel_axis = 3
+        else:
+            channel_axis = 1
+
+        y = keras.layers.TimeDistributed(
```

Comment:
Hi, @JihongJu, I added a `temporal` module to keras-resnet. You should be able to use `keras_resnet.block.temporal.basic`, `keras_resnet.block.temporal.bottleneck`, and `keras_resnet.block.temporal._shortcut`.